tx/textual/valuerenderer/message.go

```diff
@@ -159,3 +257,59 @@ func (mr *messageValueRenderer) Parse(ctx context.Context, screens []Screen) (pr
 
 	return protoreflect.ValueOfMessage(msg), nil
 }
+
+func (mr *messageValueRenderer) parseRepeated(ctx context.Context, screens []Screen, l protoreflect.List, vr ValueRenderer) error {
+
+	// <int> <field_kind>
+	headerRegex := *regexp.MustCompile(`(\d+) .+`)
```

Comment:
Should parse check the <field_kind> part of the screen too? We would need to pass `fd protoreflect.FieldDescriptor` as an arg too.

Maybe it's fine for Parse to not be that strict.

Reply:
What's the criteria for `<field_kind>`? We should perhaps add it into the regex. Also we need to ensure that the regex begins with an integer or permit spaces but essentially perhaps this
```go
headerRegex := regexp.MustCompile(`^\s*(\d+) (\w+|\d+)`)
```
and no need to dereference the result from regexp.MustCompile.

Reply:
The field kind is documented in Annex 1 of ADR 50:

field_kind:
- if the type of the repeated field is a message, field_kind is the message name
- if the type of the repeated field is an enum, field_kind is the enum name
- in any other case, field_kind is the protobuf primitive type (e.g. "string" or "bytes")